src/test/java/com/saebyeok/saebyeok/acceptance/LikeAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+package com.saebyeok.saebyeok.acceptance;
+
+import com.saebyeok.saebyeok.dto.ArticleResponse;
+import com.saebyeok.saebyeok.dto.CommentResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LikeAcceptanceTest extends AcceptanceTest {
+    private static final long COMMENT_ID = 1L;
+
+    /**
+     * Feature: 공감(좋아요) 기능
+     * Scenario: 로그인 한 회원은 게시글과 댓글에 공감(좋아요)을 표현할 수 있다.
+     * <p>
+     * given 공감이 없는 게시글과 댓글이 등록되어있다.
+     * when 회원이 게시글을 공감한다.
+     * then 해당 게시글에 공감이 추가된다.
+     * <p>
+     * when 이미 공감한 글을 공감 취소한다.
+     * then 해당 게시글에 적용된 공감이 제거된다.
+     * <p>
+     * when 회원이 댓글을 공감한다.
+     * then 해당 댓글에 공감이 추가된다.
+     * <p>
+     * when 이미 공감한 댓글을 공감 취소한다.
+     * then 해당 댓글에 적용된 공감이 제거된다.
+     **/
+    @DisplayName("게시글과 댓글에 공감(좋아요)을 표현할 수 있다.")
+    @Test
+    void manageLike() {
+        // given 공감이 없는 게시글과 댓글이 등록되어있다.
+        createArticle(ARTICLE_CONTENT, EMOTION_ID, SUB_EMOTION_IDS, true);
+        createComment(ARTICLE_ID, COMMENT_CONTENT);
+
+        // when 회원이 게시글을 공감한다.
+        likeArticle(ARTICLE_ID);
+
+        // then 해당 게시글에 공감이 추가된다.
+        ArticleResponse LikedArticle = readArticle(ARTICLE_ID);
+        assertThat(LikedArticle.getLikesCount()).isEqualTo(1);
```

Comment:
앗 변수명 첫 글자가 대문자! **Line 50-51, 57-58, 64-65**도 확인해 주세용~
```suggestion
        ArticleResponse likedArticle = readArticle(ARTICLE_ID);
        assertThat(likedArticle.getLikesCount()).isEqualTo(1);
```

Reply:
엇...ㅎㅎ 이럴수가 ㅎㅎ 수정했습니다!